code/modules/admin/verbs/deathsquad.dm

```diff
@@ -21,17 +21,15 @@ GLOBAL_VAR_INIT(deathsquad_sent, FALSE)
 	log_admin("[key_name_admin(proccaller)] has started to spawn a DeathSquad.")
 	alert("This 'mode' will go on until everyone is dead or the station is destroyed. You may also admin-call the evac shuttle when appropriate. Spawned commandos have internals cameras which are viewable through a monitor inside the Spec. Ops. Office. The first one selected/spawned will be the team leader.")
 
-	var/mission
-	mission = sanitize(copytext(input(src, "Please specify which mission the Deathsquad shall undertake.", "Specify Mission", "",), 1, MAX_MESSAGE_LEN))
+	var/mission = sanitize(copytext(input(src, "Please specify which mission the Deathsquad shall undertake.", "Specify Mission", "",), 1, MAX_MESSAGE_LEN))
 	if(!mission)
 		if(alert("Error, no mission set. Do you want to exit the setup process?",, "Yes", "No") == "Yes")
 			message_admins("[key_name_admin(proccaller)] cancelled their Deathsquad.")
 			log_admin("[key_name(proccaller)] cancelled their Deathsquad.")
 			return
 
-	var/commando_number
 	var/is_leader = TRUE // set to FALSE after leader is spawned
-	commando_number = clamp(input(src, "How many Deathsquad Commandos would you like to send? (Recommended is 6, Max is [MAX_COMMANDOS]])", "Specify Commandos") as num|null, 1, MAX_COMMANDOS)
+	var/commando_number = clamp(input(src, "How many Deathsquad Commandos would you like to send? (Recommended is 6, Max is [MAX_COMMANDOS]])", "Specify Commandos") as num|null, 1, MAX_COMMANDOS)
 	if(!commando_number)
```

Comment:
`clamp()`ing this will never make `commando_number` be null, so the `if` would always fail.
I guess the clamping should be moved to after the `if` body